src/schnorrsig.rs

```diff
@@ -211,6 +211,12 @@ impl KeyPair {
         }
     }
 
+    /// Serialize the key pair as a secret key byte value
+    #[inline]
+    pub fn serialize_sec<V: Verification>(&self, secp: &Secp256k1<V>) -> [u8; constants::SECRET_KEY_SIZE] {
+        *SecretKey::from_keypair(secp, self).as_ref()
```

Comment:
This doesn't really require a context, See https://github.com/rust-bitcoin/rust-secp256k1/pull/306#pullrequestreview-687898295

Reply:
fixed